assets/lift_inspector/acquia_lift_inspector.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * @file acquia_lift_inspector.js
+ */
+(function ($, Drupal) {
+  'use strict';
+
+  Drupal.behaviors.acquia_lift = {
+    attach: function (context, settings) {
+
+      function show_inspector_modal() {
+        var $modal_element, modal_template;
+
+        $('#lift-inspector').remove();
+
+        //Get the Profile Manager URL from the config settings
+        var accountSubDomain = drupalSettings.acquia_lift.acquia_lift_inspector.profile_manager_url;
+
+        if (accountSubDomain) {
```

Comment:
AFAICT, but we should ask Steve. In the case there is no prefix, I have accounted for that in the docs.